app/src/main/java/com/infomaniak/mail/ui/main/thread/ThreadFragment.kt

```diff
@@ -182,25 +170,21 @@ class ThreadFragment : Fragment() {
         changeToolbarColorOnScroll(toolbar, messagesListNestedScrollView) { color ->
             appBar.backgroundTintList = ColorStateList.valueOf(color)
         }
-
-        updateUi(folderRole)
     }
 
-    private fun initAdapter() = with(binding.messagesList) {
+    private fun setupAdapter() = with(binding.messagesList) {
         adapter = ThreadAdapter(
             shouldLoadDistantResources = shouldLoadDistantResources(),
             onContactClicked = { contact ->
-                safeNavigate(ThreadFragmentDirections.actionThreadFragmentToDetailedContactBottomSheetDialog(contact))
+                mainViewModel.detailedContactArgs.value = contact
```

Comment:
Same comment than the previous `downloadAttachmentsArgs` one. And actually the comment works probably for all of them

Reply:
Answered here : https://github.com/Infomaniak/android-kMail/pull/1573?w=1#discussion_r1425256192